docs/openclaw/index.md

```
@@ -1,4 +1,5 @@
-# How to use the Mammouth API in OpenClaw
+Information about OpenClaw
+#How to use the Mammouth API in OpenClaw
```

markdownlint-cli2 (0.21.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)